### PR TITLE
[release-4.10] OCPBUGS-1440: fix install plugins script

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: 2/contrib/jenkins/install-plugins.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -eu
+#! /bin/bash -eu
```
Comment (akram) [inline 2/contrib/jenkins/install-plugins.sh:1]: is that a typo ?

Comment (coreydaley) [reply 2/contrib/jenkins/install-plugins.sh]: Yep! Fixed.

**Pull request merged**